html/janus.js

```diff
@@ -1800,9 +1800,7 @@ function Janus(gatewayCallbacks) {
 				// For Chrome versions before 72, we force a plan-b semantic, and unified-plan otherwise
 				pc_config["sdpSemantics"] = (Janus.webRTCAdapter.browserDetails.version < 72) ? "plan-b" : "unified-plan";
 			}
-			var pc_constraints = {
-				"optional": [{"DtlsSrtpKeyAgreement": true}]
-			};
+			var pc_constraints = {};
 			if(ipv6Support) {
 				pc_constraints.optional.push({"googIPv6":true});
 			}
```

Comment:
The code below, "Any custom constraints to add?", is also unnecessary now that there are no other constraints available than the two deprecated ones above: https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/renderer/modules/peerconnection/goog_media_constraints.idl

The rest of the constraints have already been removed and are necessarily NO-OPs on up-to-date versions of Chromium. The correct way to construct an RTCPeerConnection is to not provide a second argument (pc_constraints)

Reply:
@lminiero FYI

Reply:
Awesome! (You can delete the entire pc_constraints variable when you do this)